pkg/controller/alb-controller.go

```diff
@@ -244,6 +245,15 @@ func (ac *ALBController) Info() *ingress.BackendInfo {
 func (ac *ALBController) ConfigureFlags(pf *pflag.FlagSet) {
 	pf.StringVar(&ac.clusterName, "clusterName", os.Getenv("CLUSTER_NAME"), "Cluster Name (required)")
 
+	rs, _ := strconv.ParseBool(os.Getenv("ALB_CONTROLLER_RESTRICT_SCHEME"))
```

Comment:
We should do something with this error?

Perhaps we `panic()` stating something along the lines `ALB_CONTROLLER_RESTRICT_SCHEME` must be true or false. Instead was `X`.